### PR TITLE
Gamedb : ADD fixes for various games 

### PR DESCRIPTION
**Pull request by Mrlinkwii**
### Description of Changes

add 'GIFFIFOHack' to 'FIFA 2005
### Rationale behind Changes

The hack helps the game 
### Suggested Testing Steps

look at CI

### CODE REVIEW

**Review by JordanTheToaster — APPROVED** [review]

File: bin/GameIndex.yaml [hunk 1]
```diff
@@ -7072,6 +7072,11 @@ SLED-52597:
 SLED-52875:
   name: "Sega Superstars [Demo]"
   region: "PAL-E"
+SLED-52890:
+  name: "FIFA 2005 [Demo]"
+  region: "PAL"
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:7077]:
```suggestion
  region: "PAL-M7"
```

File: bin/GameIndex.yaml [hunk 2]
```diff
@@ -7072,6 +7072,11 @@ SLED-52597:
...
+  gameFixes:
+    - GIFFIFOHack # Partily fixed grpahical issues also needs EE cycle rate + 3.
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:7079]:
```suggestion
    - GIFFIFOHack # Partially fixes graphical issues also needs EE cycle rate + 3.
```

Comment (RedDevilus) [reply bin/GameIndex.yaml:7079]: Copy to the rest of the comments

File: bin/GameIndex.yaml [hunk 3]
```diff
@@ -19601,6 +19625,16 @@ SLPM-60262:
 SLPM-60265:
   name: "10th Anniversary PlayStation & PlayStation 2 All-Soft Catalogue Special SaveData Collection [PS2 Disc]"
   region: "NTSC-J"
+SLPM-60270:
+  name: "D1 Professional Drift Grand Prix Series 2005 [Trial]"
+  region: "NTSC-J"
+  clampModes:
+    eeClampMode: 3 # Fixes black screen
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:19632]:
```suggestion
    eeClampMode: 3 # Fixes black screen.
```

File: bin/GameIndex.yaml [hunk 4]
```diff
@@ -19601,6 +19625,16 @@ SLPM-60262:
...
+SLPM-60271:
+  name: "D1 Professional Drift Grand Prix Series 2005 [Trial]"
+  region: "NTSC-J"
+  clampModes:
+    eeClampMode: 3 # Fixes black screen
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:19637]:
```suggestion
    eeClampMode: 3 # Fixes black screen.
```

File: bin/GameIndex.yaml [hunk 5]
```diff
@@ -25070,6 +25104,8 @@ SLPM-66143:
 SLPM-66144:
   name: "D1 Grand Prix 2005"
   region: "NTSC-J"
+  clampModes:
+    eeClampMode: 3 # Fixes black screen
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:25108]:
```suggestion
    eeClampMode: 3 # Fixes black screen.
```

File: bin/GameIndex.yaml [hunk 6]
```diff
@@ -37027,6 +37063,8 @@ SLUS-21051:
   name: "FIFA 2005"
   region: "NTSC-U"
   compat: 5
+  gameFixes:
+    - GIFFIFOHack # Partially fixes graphical issues also needs EE cycle rate +
```
Comment (RedDevilus) [inline bin/GameIndex.yaml:37067]:
```suggestion
    - GIFFIFOHack # Partially fixes graphical issues also needs EE cycle rate + 3.
```

**Review by RedDevilus — CHANGES_REQUESTED** [review]

File: bin/GameIndex.yaml [hunk 7]
```diff
@@ -6738,6 +6762,8 @@ SCUS-97627:
 SCUS-97633:
   name: "Buzz! The Hollywood Quiz [Game Only]"
   region: "NTSC-U"
+  gameFixes:
+    - EETimingHack
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:6766]: missing comment

Comment (Mrlinkwii) [reply bin/GameIndex.yaml]: Done :+1:

File: bin/GameIndex.yaml [hunk 8]
```diff
@@ -12011,18 +12042,28 @@ SLES-52558:
 SLES-52559:
   name: "FIFA 2005"
   region: "PAL-E"
+  gameFixes:
+    - GIFFIFOHack # Partially fixes graphicalissues also needs EE cycle rate + 3.
```
Comment (refractionpcsx2) [inline bin/GameIndex.yaml:12046]: graphicalissues

**Pull request merged**